examples/test_feature_example_group_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe "test feature example group" do
+  feature "test.feature"
```

Comment:
Both have their adv. and disadv.

rutabaga ways can promote unique of steps, so you dont need to worry about clashing tags etc. but if the feature file relocated or renamed, this will break

while turnip's way will work even if the feature / steps files changed location. but it could be a problem with clashing tag names etc